data/navbar.js

```diff
@@ -11,6 +11,10 @@ const NavbarData = [
     name: 'Activities',
     link: '/activities',
   },
+  {
+    name: 'Sponsors us',
+    link: '/sponsors',
+  },
   {
     name: 'Startups',
     link: '/startups',
```

Comment:
Comment the startup section from the navbar till the drop down menu feature is developed